Makefile.am

```diff
@@ -27,6 +27,8 @@ TESTS= tests/display.scm
 
 AM_TESTS_ENVIRONMENT = abs_top_srcdir="$(abs_top_srcdir)"
 
+README: README.org
+
```

Comment:
no need add it for makefile, just use `ln -s README.org README` ,likes b165499

Reply:
Hm... I hadn't have local (lisp/wayland/README) symlinked with the "top level one" and build system would throw an error. Maybe git doesn't track symlinks by default?

Reply:
The compilation system does not depend on git, `README: README.org` is tell Make `README` depend by `README.org`, but you no write how to create `README`, so Make cann't create `README`.